### PR TITLE
Updated management SDK for Azure Attestation Service to latest swagger spec with Location and Tags support

### PR DESCRIPTION
**Pull request by gkostal**
**What**
---
 This is the update to the .NET management SDK for the Azure Attestation Service based on the swagger changes made in this PR:

https://github.com/Azure/azure-rest-api-specs/pull/8478

**Tests**
---
New tests have been added, recordings have been generated and all tests are passing in playback mode.

### CODE REVIEW

File: sdk/attestation/Microsoft.Azure.Management.Attestation/tests/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -35,5 +35,5 @@
 // You can specify all the values or you can default the Build and Revision Numbers 
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
-[assembly: AssemblyVersion("0.10.0.0")]
-[assembly: AssemblyFileVersion("0.10.0.0")]
+[assembly: AssemblyVersion("0.11.0.0")]
```
Comment (isra-fel) [inline sdk/attestation/Microsoft.Azure.Management.Attestation/tests/Properties/AssemblyInfo.cs]: Updating assembly info of the test project is not required, as it's not published anywhere.

**Review by isra-fel — COMMENTED** [review]
Hey @gkostal , I assume this file `sdk/attestation/Microsoft.Azure.Management.Attestation/src/Generated/code-model-v1` was checked in by accident. I have deleted it.
And by the way, did you generate code by running `generate.ps1` or `autorest.cmd`?

**Review by isra-fel — APPROVED** [review]

File: sdk/attestation/Microsoft.Azure.Management.Attestation/tests/Properties/AssemblyInfo.cs (hunk 1 above)
Comment (gkostal) [reply sdk/attestation/Microsoft.Azure.Management.Attestation/tests/Properties/AssemblyInfo.cs]: Makes sense.  I've inherited this code and was updating all existing 0.10.0.0 version numbers to 0.11.0.0.  It's OK to leave as 0.11.0.0 right?

**Comment by isra-fel** [comment]
Management place CI passed. Merging the PR.

**Pull request merged**